system-x/services/jira/src/main/java/software/tnb/jira/account/JiraAccount.java

```diff
@@ -7,10 +7,6 @@ public class JiraAccount implements Account, WithId {
     private String jira_url;
     private String username;
     private String password;
-    private String private_key;
-    private String access_token;
-    private String consumer_key;
-    private String verification_code;
```

Comment:
these were generated and we don't need them? or is there other reason why they are removed?

Reply:
I searched for usages and I didn't find anything, moreover I updated the credentials, those values are outdated, IMO we don't have any reason to keep them.

Reply:
makes sense, thanks!